Tests/SwiftLintFrameworkTests/UnusedOptionalBindingRuleTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UnusedOptionalBindingRuleTests.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 05/01/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+@testable import SwiftLintFramework
+import XCTest
+
+class UnusedOptionalBindingRuleTests: XCTestCase {
+
+    func testDefaultConfiguration() {
+        let baseDescription = UnusedOptionalBindingRule.description
```

Comment:
Maybe we could make a lens for `RuleDescription` to simplify this? See http://chris.eidhof.nl/post/lenses-in-swift/